src/index.html

```diff
@@ -1,16 +1,53 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form class="big_form"
+      action="https://www.figma.com/file/kf3AWulK9elrNk34wtpjPw/Airbnb-Search-bar?node-id=0%3A1"
+      method="post"
+      data-qa="big"
+    >
+      <label >
+        <img src="images/Search.svg"
+        alt="Search icon"
+        class="img_big">
+        <input
+        type="search"
+        class="field_big"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+
+    </form>
+    <form class="small_form"
+      action="https://www.figma.com/file/kf3AWulK9elrNk34wtpjPw/Airbnb-Search-bar?node-id=0%3A1"
+      method="post"
+      data-qa="small"
+    >
+      <label>
+        <img src="images/Search.svg"
+        alt="Search icon"
+        class="img_small"
+        >
+        <input
+        type="search"
+        class="field_small"
+        placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
```

Comment:
The same here, check 4th point from the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)